This sets -march=core2 on osx x86_64. We want to let our activation scripts do their job.
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 718edeb..d234762 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,10 +74,10 @@ if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm" OR ${CMAKE_SYSTEM_PROCESSOR} MATCHES
     endif()
 else()
     message(STATUS "x86 detected")
-    if (APPLE AND CT_INSTRUCTIONS STREQUAL "basic")
-        # Universal binary.
-        set(CMAKE_OSX_ARCHITECTURES "arm64;x86_64" CACHE STRING "" FORCE)
-    endif()
+    # if (APPLE AND CT_INSTRUCTIONS STREQUAL "basic")
+    #     # Universal binary.
+    #     set(CMAKE_OSX_ARCHITECTURES "arm64;x86_64" CACHE STRING "" FORCE)
+    # endif()
 
     if (MSVC)
         if (CT_INSTRUCTIONS STREQUAL "avx2")
